src/actions/game.ts

```diff
@@ -1,5 +1,8 @@
+import { GameState } from 'src/reducers/states';
 import { action } from 'typesafe-actions';
-
 import * as actionTypes from './actionTypes';
 
+export const fetchTestStories = () => action(actionTypes.FETCH_TEST_STORIES);
```

Comment:
Error when I try to start the server:
```(5,1): Module './game' has already exported a member named 'fetchTestStories'. Consider explicitly re-exporting to resolve the ambiguity.```

There is a second `fetchTestStories` export coming from `src/actions/material.ts`, I'm guessing one of these needs to be removed.

Reply:
sure, gimme a moment

Reply:
Done.